pkg/controllers/nodeclaim/disruption/drift.go

```diff
@@ -48,21 +49,22 @@ type Drift struct {
 
 func (d *Drift) Reconcile(ctx context.Context, nodePool *v1beta1.NodePool, nodeClaim *v1beta1.NodeClaim) (reconcile.Result, error) {
 	hasDriftedCondition := nodeClaim.StatusConditions().GetCondition(v1beta1.Drifted) != nil
+	reconcilieLog := log.FromContext(ctx).V(1)
```

Comment:
I don't love this pattern, personally. I think we should just call `V(1)` when it's needed. If we call it everwhere in this function consistently, so be it